lib/migration.js

```diff
@@ -461,8 +461,8 @@ function mixinMigration(MySQL, mysql) {
 
   function columnType(p, defaultType) {
     var dt = defaultType;
-    if (p.dataType) {
-      dt = String(p.dataType);
```

Comment:
I'm worried about backwards compatibility here... Maybe do an "else if" keeping this code.